tensorflow/go/session.go

```diff
@@ -65,6 +65,53 @@ func NewSession(graph *Graph, options *SessionOptions) (*Session, error) {
 	return s, nil
 }
 
+type Device struct {
+	Name, Type  string
+	MemoryLimit int64
```

Comment:
Perhaps call this `MemoryLimitBytes` to make the units clearer?

Reply:
Done, thanks.